README-development.md

```diff
@@ -0,0 +1,12 @@
+
+## Making a Release
+
+* `npm version patch`
```

Comment:
or `npm version major` or `npm version minor`

No `npm install` or `git commit` or `git tag` is needed: `npm version` does all of that.